src/app/search/search.component.ts

```diff
@@ -103,10 +98,24 @@ export class SearchComponent implements OnInit {
    */
   constructor(private providerService: ProviderService) {
     this._client = new Client({
-      host: 'http://10.11.9.73:8080/api/ga4gh/v1/extended',
+      host: Dockstore.API_URI + '/api/ga4gh/v1/extended',
       apiVersion: '2.4',
       log: 'trace'
     });
+    const shard_size = 10000;
+    const body = bodybuilder()
+      .aggregation('terms', '_type', { size: shard_size }, '_type')
+      .aggregation('terms', 'registry', { size: shard_size }, 'registry')
+      .aggregation('terms', 'private_access', { size: shard_size }, 'private_access')
+      .aggregation('terms', 'tags.verified', { size: shard_size }, 'tags_verified')
+      .aggregation('terms', 'author', { size: shard_size }, 'author')
+      .aggregation('terms', 'namespace', { size: shard_size }, 'namespace')
+      .aggregation('terms', 'labels.value', { size: shard_size }, 'labels_value')
+      .aggregation('terms', 'tags.verifiedSource', { size: shard_size }, 'tags_verifiedSource')
+      .query('match_all', {})
+      .size(500);
```

Comment:
For later, probably don't want to retrieve 500, probably better to grab a page worth at a time